packages/office-ui-fabric-react/src/components/Callout/CalloutContent.tsx

```diff
@@ -18,6 +18,9 @@ import { getRelativePositions, IPositionInfo, IPositionProps, getMaxHeight } fro
 import { Popup } from '../../Popup';
 import * as stylesImport from './Callout.scss';
 import { AnimationClassNames, mergeStyles } from '../../Styling';
+import {
+  CommandButton
```

Comment:
Hmm. It seems like you would just have some frame to render inside of the callout. Callout is agnostic of content, but this is violating that principle. Instead of adding special cases to Callout, could we instead abstract this in a sort of layout component which can abstract the layout you're shooting for?

For example:

```tsx
<Callout>
  <FancyLayout
    onRenderHeader={ () => <FancyHeader /> }
    onRenderContent={ () => <div>I am content</div> }
    onRenderFooter={ () => <FancyFooter /> }
  />
</Callout>
```

Reply:
We are using the ComboBox, is it possible to insert this from the ComboBox into the Callout?

Reply:
@dzearing - Hi!  Not sure if you saw my previous comment, but I am using a ComboBox and passing this in as a prop to the Callout.  I don't know if there is a way to add it to the Callout Component from the ComboBox other than using the prop I created.

Reply:
I experimented with passing a button element in a prop from the ComboBox to the Callout Component in ComboBox.tsx, but this doesn't add the element to the right place.  It adds it within .ms-Callout-main instead of outside it.